lib/dslash_policy.cuh

```diff
@@ -2018,11 +1950,13 @@ public:
         setPolicyTuning(true);
       }
       dslash_policy_init = true;
-   }
+
+      apply(device::get_default_stream());
+    }
 
    virtual ~DslashPolicyTune() { setPolicyTuning(false); }
 
-   void apply(const qudaStream_t &stream) {
+   void apply(const qudaStream_t &) {
```

Comment:
Just wondering: If this is no longer exposed b/c it is called when constructing the policy: make it private?

Reply:
ae1ba594b